config.yaml

```diff
@@ -142,3 +142,13 @@ options:
       Loads example data source with corresponding data sets and charts for testing.
     type: boolean
     default: False
+  html-sanitization:
```

Comment:
This would put this config out of convention with the others - there are a lot of flags that follow the convention of matching the name of the superset config parameter and default to true. If I change this I would change it across all of these in a separate PR.